finagle-mux/src/main/scala/com/twitter/finagle/mux/transport/Message.scala

```diff
@@ -476,6 +478,7 @@ private[finagle] object Message {
       // TODO: technically we should probably check for duplicate
       // keys, but for now, just pick the latest one.
       key match {
+        // TODO: support traceIdHigh
```

Comment:
`Treq` is dead to us: its only used for backward compatibility so I don't think a TODO is necessary. A note that we're not going to support 128-bit trace id's in `Treq` is enough. If someone upgrades, which they'd need to do to support the 128-bit trace id's anyway, they are going to be using `Tdispatch/Rdispatch` which should get 128-bit via this PR. 

Someone should correct me if this is not right.

Reply:
Agreed.